geps/gep-3155/index.md

```diff
@@ -0,0 +1,166 @@
+# GEP-3155: Complete Backend mutual TLS Configuration
+
+* Issue: [#3155](https://github.com/kubernetes-sigs/gateway-api/issues/3155)
+* Status: Implementable
+
+## TLDR
+
+This GEP aims to complete the configuration required for Backend mutual TLS in Gateway
+API. This includes the following new capabilities:
+
+1. Configuration for the client certificate Gateways should use when connecting
+   to Backends
+1. Ability to specify SANs on BackendTLSPolicy
+1. Add TLS options to BackendTLSPolicy to mirror TLS config on Gateways
+
+## Goals
+
+* Add sufficient configuration that basic mutual TLS is possible between Gateways and
+  Backends
+* Enable the optional use of SPIFFE for Backend mutual TLS
+
+## Non-Goals
+
+* Define how automatic mTLS should be implemented with Gateway API
+
+## Introduction
+
+This is a wide ranging GEP intending to cover three additions to the API that all
+have a shared goal - enabling backend mutual TLS with Gateway API. Although this
+specific GEP focuses on manual configuration across the board, the hope is that
+it will also enable higher level automation to simplify this process for users.
+
+## API
+
+### Client Certs on Gateways
+
+A key requirement of mutual TLS is that the Gateway can provide a client cert to the
+backend. This adds that configuration to both Gateway and Service (via 
+BackendTLSPolicy).
+
+#### Gateway-level (Core support)
```

Comment:
This is missing an API example - seems to be suggesting the addition of a new field under the Gateway `spec` stanza https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1.Gateway?

Reply:
Yes, I think this should have a quick sketch of what we need to add to Gateway.